test/integration/evictions/evictions_test.go

```diff
@@ -216,14 +221,16 @@ func TestConcurrentEvictionRequests(t *testing.T) {
 		_, err := clientSet.Core().Pods(ns.Name).Get(podName, metav1.GetOptions{})
 		if !errors.IsNotFound(err) {
```

Comment:
might it be possible to check that the eviction has been completed inside of the goroutines spawned on line 177. The variable name `doneCh` may be considered a bit misleading given that we haven't checked that the pod was actually evicted before saying that we're done. If the final check that a pod was actually evicted was moved into the goroutine where we requested the pod be evicted this for loop could possibly be removed